src/parameterizations/vertical/MOM_CVMix_KPP.F90

```diff
@@ -110,6 +114,7 @@ module MOM_CVMix_KPP
   logical :: LT_K_Enhancement          !< Flags if enhancing mixing coefficients due to LT
   integer :: LT_K_Shape                !< Integer for constant or shape function enhancement
   integer :: LT_K_Method               !< Integer for mixing coefficients LT method
+  real    :: KPP_CVt2                  !< Parameter for Stokes MOST convection entrainment
```

Comment:
Please add a description of the units of KPP_CVt2, perhaps `[nondim]`.

Reply:
Addressed in [a4d2173](https://github.com/NCAR/MOM6/pull/344/commits/a4d2173a5cf1393caa351d47ca658be4a581405f)